src/main/features/response/helpers/taskListBuilder.ts

```diff
@@ -58,13 +58,24 @@ export class TaskListBuilder {
     const externalId: string = claim.externalId
     const tasks: TaskListItem[] = []
 
-    tasks.push(
-      new TaskListItem(
-        'Choose a response',
-        Paths.responseTypePage.evaluateUri({ externalId: externalId }),
-        OweMoneyTask.isCompleted(draft)
+    if (!ClaimFeatureToggles.areAdmissionsEnabled(claim)
+      && (draft.isResponsePartiallyAdmitted() || draft.isResponseFullyAdmitted())) {
+      tasks.push(
+        new TaskListItem(
+          'Choose a response',
+          Paths.responseTypePage.evaluateUri({ externalId: externalId }),
+          false
+        )
```

Comment:
Don't we want to be hiding the check and send instead?

Reply:
No I'm afraid not after having a discussion with @robertparkinson

Reply:
@rattur cleaner solution would be when the users hit a paper handoff page from choose a response then delete the saved value for the choose a response page  from the draft object.

Reply:
Or make check-and-send redirect to the hand-off page?

Reply:
So then we could instead _not_ save the response option if we redirect to the hand-off page?

Reply:
that may work, worth testing....It will be even smaller change if everything works